lib/webauthn/attestation_statement/base.rb

```diff
@@ -7,6 +7,8 @@ module AttestationStatement
     class Base
       class NotSupportedError < Error; end
 
+      AAGUID_EXTENSION_OID = "1.3.6.1.4.1.45724.1.1.4"
```

Comment:
Putting this here since TPM attestation (which I plan to work on in the next few weeks) needs this too.